frontend/pages/sign-up.tsx

```diff
@@ -0,0 +1,299 @@
+          </div>
+        )}
+        <div className="flex gap-4">
+          <div className="w-full">
+            <label htmlFor="first-name">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="First name" id="pONqz8" />
+              </p>
+              <Input
+                type="text"
+                name="firstName"
+                id="first-name"
+                placeholder="Insert your name"
+                aria-describedby="first-name-error"
+                register={register}
+                registerOptions={{
+                  required: intl.formatMessage({
+                    defaultMessage: 'You need to enter a name.',
+                    id: 'Pl5xI4',
+                  }),
+                }}
+              />
+            </label>
+            {errors.firstName && (
+              <p
+                id="first-name-error"
+                className="mt-1 ml-2 font-sans text-xs text-red font-regular"
+              >
+                {errors.firstName.message}
+              </p>
+            )}
+          </div>
+          <div className="w-full">
+            <label htmlFor="last-name">
+              <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+                <FormattedMessage defaultMessage="Last name" id="txUL0F" />
+              </p>
+              <Input
+                type="text"
+                name="lastName"
+                id="last-name"
+                placeholder="Insert your last name"
+                aria-describedby="last-name-error"
+                register={register}
+                registerOptions={{
+                  required: intl.formatMessage({
+                    defaultMessage: 'You need to enter a last name.',
+                    id: 'zPMPr9',
+                  }),
+                }}
+              />
+            </label>
+            {errors.lastName && (
+              <p id="last-name-error" className="mt-1 ml-2 font-sans text-xs text-red font-regular">
+                {errors.lastName.message}
+              </p>
+            )}
+          </div>
+        </div>
+        <div className="w-full">
+          <label htmlFor="email">
+            <p className="mb-2.5 mt-4.5 font-sans text-sm font-semibold text-gray-800">
+              <FormattedMessage defaultMessage="Email" id="sy+pv5" />
+            </p>
+            <Input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Insert your email"
+              aria-describedby="email-error"
+              register={register}
+              registerOptions={{
+                required: intl.formatMessage({
+                  defaultMessage: 'You need to enter your email',
+                  id: 'NioTuC',
+                }),
+                validate: (value) =>
```

Comment:
I'm not sure why, but if you enter some random text, the error is not displayed below the input. Try with putting just “clement” for example.

![The email field is invalid but no error is displayed below it](https://user-images.githubusercontent.com/6073968/159652841-b2146566-033b-4e5f-8dbd-9270346bfeb9.png)